src/Utilities/ParsedCss.php

```diff
@@ -0,0 +1,284 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Pelago\Emogrifier\Utilities;
+
+/**
+ * Parses and stores a CSS document from a string of CSS, with methods to return the CSS in parts or as data structures.
+ *
+ * @internal
+ */
+class ParsedCss
```

Comment:
Thinking about the name: I don't think that the `Parsed` needs to be part of the class name (even if it might make sense for instance variable names to distinguish the parsed CSS and the raw CSS).

Reply:
OK, will change, either to `Css` or `CssDocument`.  Sabberworm uses `CssDocument` which I like, though would mean their FQN would have to be `use`d as something else.

Reply:
We can `use` it with an alias:

```php
use Vendor\App\Magic as SabberwormMagic;
```

Reply:
Yep, that's what I was thinking :)